e2e/utils/utils.go

```diff
@@ -0,0 +1,173 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"context"
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sigs.k8s.io/e2e-framework/klient"
+	"sigs.k8s.io/e2e-framework/third_party/helm"
+	"time"
+)
+
+// EXPORTED:
+
+var IstioInjectionLabel = map[string]string{
+	"istio-injection": "enabled",
+}
+
+func InstallCurl() error {
+	cmd := exec.Command("kubectl", "-n", "test", "apply", "-f", "curl.yaml")
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to execute command. %v, %s", err, out)
+	}
+	return nil
+}
+
+func LoadDockerImagesToCluster(cluster, tag string) error {
+	if err := LoadDockerImageToCluster(cluster, fmt.Sprintf("ghcr.io/apiclarity/apiclarity:%v", tag)); err != nil {
+		return fmt.Errorf("failed to load docker image to cluster: %v", err)
+	}
+	if err := LoadDockerImageToCluster(cluster, fmt.Sprintf("ghcr.io/apiclarity/kong-plugin:%v", tag)); err != nil {
+		return fmt.Errorf("failed to load docker image to cluster: %v", err)
+	}
+	if err := LoadDockerImageToCluster(cluster, fmt.Sprintf("ghcr.io/apiclarity/tyk-plugin-v3.2.2:%v", tag)); err != nil {
+		return fmt.Errorf("failed to load docker image to cluster: %v", err)
+	}
+	if err := LoadDockerImageToCluster(cluster, fmt.Sprintf("ghcr.io/apiclarity/passive-taper:%v", tag)); err != nil {
+		return fmt.Errorf("failed to load docker image to cluster: %v", err)
+	}
+
+	return nil
+}
+
+func LoadDockerImageToCluster(cluster, image string) error {
```

Comment:
why not to use https://github.com/kubernetes-sigs/e2e-framework/blob/main/support/kind/kind.go#L207?

Reply:
Didn't work for some reason